### PR TITLE
Add test for convert.py

### PR DESCRIPTION
**Pull request by RandySheriffH**
Create test cases to cover convert.py.

### CODE REVIEW

File: tests/test_convert.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
+import sys
```
Comment (guschmue) [inline tests/test_convert.py]: copy a standard header from the other unit tests, like copyright and docstr

File: tests/test_convert.py [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+import unittest
+import os
```
Comment (guschmue) [inline tests/test_convert.py]: pep8: should be sorted

File: tests/test_convert.py [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+from tf2onnx import convert
```
Comment (guschmue) [inline tests/test_convert.py]: pep8: 1 line between sys imports and local imports

File: tests/test_convert.py [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+
+def run_test_case(args):
```
Comment (guschmue) [inline tests/test_convert.py]: pep8: 2 lines

File: tests/test_convert.py [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+    sys.argv = args
+    convert.main()
+    ret = os.path.exists(args[-1])
+    if ret == True:
+        os.remove(args[-1])
+    return ret
+
```
Comment (guschmue) [inline tests/test_convert.py]: pep8: 2 lines

File: tests/test_convert.py [hunk 6]
```diff
@@ -0,0 +1,23 @@
...
+class Tf2OnnxConvertTest(unittest.TestCase):
+    def test_convert_saved_model(self):
```
Comment (guschmue) [inline tests/test_convert.py]: +1 line

File: tests/test_convert.py [hunk 7]
```diff
@@ -0,0 +1,23 @@
...
+        self.assertTrue(run_test_case(['', '--saved-model', 'tests/models/regression/saved_model', '--output', 'converted_saved_model.onnx']))
```
Comment (guschmue) [inline tests/test_convert.py]: line too long

File: tests/test_convert.py [hunk 8]
```diff
@@ -0,0 +1,23 @@
...
+    def test_convert_graphdef(self):
+        self.assertTrue(run_test_case(['', '--input', 'tests/models/regression/graphdef/frozen.pb', '--inputs', 'X:0', '--outputs', 'pred:0', '--output', 'converted_graphdef.onnx']))
+    def test_convert_checkpoint(self):
+        self.assertTrue(run_test_case(['', '--checkpoint', 'tests/models/regression/checkpoint/model.meta', '--inputs', 'X:0', '--outputs', 'pred:0', '--output', 'converted_checkpoint.onnx']))
+
+if __name__ == '__main__':
+    unittest.main()
```
Comment (guschmue) [inline tests/test_convert.py]: last cr missing

File: tests/test_convert.py (hunk 5 above)
Comment (RandySheriffH) [reply tests/test_convert.py]: fixed.

File: tests/test_convert.py (hunk 4 above)
Comment (RandySheriffH) [reply tests/test_convert.py]: fixed.

File: tests/test_convert.py (hunk 8 above)
Comment (RandySheriffH) [reply tests/test_convert.py]: fixed.

File: tests/test_convert.py (hunk 6 above)
Comment (RandySheriffH) [reply tests/test_convert.py]: done

File: tests/test_convert.py (hunk 3 above)
Comment (RandySheriffH) [reply tests/test_convert.py]: added.

File: tests/test_convert.py (hunk 2 above)
Comment (RandySheriffH) [reply tests/test_convert.py]: sorted now.

File: tests/test_convert.py (hunk 1 above)
Comment (RandySheriffH) [reply tests/test_convert.py]: copied.

**Pull request merged**